REUSE.toml

```
@@ -0,0 +1,48 @@
+version = 1
+
+[[annotations]]
+path = [
+    "src/**",
+    "driver/**"
+]
+SPDX-FileCopyrightText = "2023-2025 ESTKME TECHNOLOGY LIMITED, Hong Kong"
+SPDX-License-Identifier = "AGPL-3.0-only"
+
+[[annotations]]
+path = ["euicc/**"]
+SPDX-FileCopyrightText = "2023-2025 ESTKME TECHNOLOGY LIMITED, Hong Kong"
+SPDX-License-Identifier = "LGPL-2.1-only OR LicenseRef-ESTKME-Commercial"
+
+[[annotations]]
+path = [
+    ".github/**",
+    "scripts/**",
+    "cmake/**",
+    "CMakeLists.txt",
+    "README.md",
+    "docs/**",
+    ".editorconfig",
+    ".gitignore"
+]
```

```suggestion
path = [
    ".github/**",
    "cmake/**",
    "docs/**",
    "scripts/**",
    "README.md",
    "CMakeLists.txt",
    ".editorconfig",
    ".gitignore"
]
```

re-order file patterns

---

You broke my order...It's aux files, build systemd files, documents and config files for tool...